packages/gitmoji-changelog-core/src/index.spec.js

```diff
@@ -60,14 +60,12 @@ const secondLipstickCommit = {
 }
 
 describe('changelog', () => {
-  beforeEach(() => {
+  it('should generate changelog for next release', async () => {
     mockGroups()
-  })
 
-  it('should generate changelog in json format for next release', async () => {
-    gitSemverTags.mockImplementation((cb) => cb(null, []))
+    gitSemverTags.mockImplementation(cb => cb(null, []))
 
-    const { changes } = await generateChangelog()
+    const { changes } = await generateChangelog({ mode: 'incremental', release: 'next' })
```

Comment:
I don't see any references at `incremental` mode in the code maybe i missed something ? Should it be `update` ?

Reply:
Yes it's update :P